Source/WebCore/platform/qt/PasteboardQt.cpp

```diff
@@ -333,7 +333,8 @@ String Pasteboard::readString(const String& type)
         return data->text();
 
     QByteArray rawData = data->data(mimeType);
-    QString stringData = QTextCodec::codecForName("UTF-16")->toUnicode(rawData);
+    auto toUtf16 = QStringDecoder(QStringDecoder::Utf8);
```

Comment:
This looks wrong. `rawData` seems to be in UTF-16 already.
Not sure why the original code used QTextCodec in the first place, because `QString` can convert from UTF16 natively:
`return QString::fromUtf16(reinterpret_cast<const char16_t *>(rawData.constData()), rawData.size());`